app/build.gradle

```diff
@@ -33,6 +33,7 @@ android {
 }
 
 dependencies {
+    implementation 'com.google.android.libraries.places:places:2.6.0'
     implementation "com.github.parse-community.Parse-SDK-Android:parse:1.18.5"
     implementation "androidx.appcompat:appcompat:${appCompatVersion}"
     implementation "com.google.android.material:material:${designSupportVersion}"
```

Comment:
I like how you are using these versions, but these are both duplicated below (with higher version numbers).  Delete the ones below and update the version numbers in your other build.gradle to match the higher numbers below.